### ISSUE TITLE
Authentication

### ISSUE THREAD

**Issue by ePascalC** [ePascalC/bbp-API#17]
Review Cookie authentication on https://developer.wordpress.org/rest-api/using-the-rest-api/authentication/

and

https://github.com/jonathan-dejong/wp-api-jwt-auth

**Comment by tkserver** [comment]
Yes in my studies so far, JWT authentication can be better than cookies as it is persistent. Working well so far for me.

**Comment by ePascalC** [comment]
It seems oAuth2 will be the chosen one:
Current development: https://github.com/WP-API/OAuth2
Used on wordpress.COM : https://developer.wordpress.com/docs/oauth2/

See also:
https://make.wordpress.org/core/2017/08/23/rest-api-roadmap/
